system/services/assetManager/AssetManagerService.cfc

```diff
@@ -1870,6 +1874,14 @@ component displayName="AssetManager Service" {
 		_setTypes( types );
 	}
 
+	private string function _getAssetPath( string appBasePath="/asset/" ) {
```

Comment:
This of course should be _getAssetPath() - Will await other instructions before committing again